schema-registry-mock/src/main/java/com/bakdata/schemaregistrymock/SchemaRegistryMock.java

```diff
@@ -83,7 +89,7 @@
  * To retrieve the url of the schema registry for a Kafka Streams config, please use {@link #getUrl()}
  */
 @Slf4j
-public class SchemaRegistryMock implements BeforeEachCallback, AfterEachCallback {
+public class SchemaRegistryMock {
```

Comment:
in this class there seem to be a few methods that do not follow our guidelines regarding `final` variables. please check with a linter and update non-final vars.